src/components/informationBlock/InformationBlock.tsx

```diff
@@ -0,0 +1,237 @@
+import { Chip, Grid, Typography, Button, Alert } from '@mui/material'
+import { InformationContainer } from '@pagopa/interop-fe-commons'
+import LaunchIcon from '@mui/icons-material/Launch'
+import LockIcon from '@mui/icons-material/Lock'
+import RefreshIcon from '@mui/icons-material/Refresh'
+import { useState, useEffect } from 'react'
+import { useTranslation } from 'react-i18next'
+import { v4 as uuid } from 'uuid'
+import format from 'date-fns/format'
+
+interface MainData {
+  eserviceName: string
+  producerName: string
+  versionNumber: string
+}
+
+interface ProbingData {
+  probingEnabled: boolean
+  state: string
+  responseReceived: string
+  eserviceActive: boolean
+}
+
+interface IProps {
+  mainData: MainData
+  probingData: ProbingData
+  reloadProbingDetails: () => void
+  viewInCatalogue?: () => void
+}
+
+const blocksInfo: Array<{ title: string; blocks: Array<string> }> = [
+  {
+    title: 'Informazioni generali',
+    blocks: ['producerName', 'versionNumber', 'eserviceName'],
+  },
+  {
+    title: 'In tempo reale',
+    blocks: ['probingEnabled', 'state', 'responseReceived'],
+  },
+]
+
+export const InformationBlock: React.FC<IProps> = ({
+  mainData,
+  probingData,
+  reloadProbingDetails,
+  viewInCatalogue,
+}) => {
+  const [alert, setAlert] = useState({
+    show: false,
+    message: '',
+  })
+
+  const { t } = useTranslation(['detailsPage', 'general'])
+
+  useEffect(() => {
+    if (probingData.probingEnabled === false && probingData.state === 'n/d') {
+      setAlert({
+        show: true,
+        message: 'monitoringSystemSuspendedMessage',
+      })
+    } else if (probingData.state === 'offline' && !probingData.eserviceActive) {
+      setAlert({
+        show: true,
+        message: 'versionSuspendedMessage',
+      })
+    } else if (probingData.state === 'offline' && probingData.eserviceActive) {
+      setAlert({
+        show: true,
+        message: 'eserviceNotAnswerMessage',
+      })
+    } else {
+      setAlert({
+        show: false,
+        message: '',
+      })
+    }
+  }, [probingData])
```

Comment:
C'è un motivo per cui tutta questa logica è dentro uno `useEffect` e non direttamente in JSX?

Si potrebbe fare estrapolare in un componente a parte, in questo modo:

```typescript
function ProbingDataAlert({ probingData }: ProbingDataAlertProps) {
  const { t } = useTranslation(['detailsPage', 'general'])

  if (!probingData) return null

  let message: undefined | string

  if (probingData.probingEnabled === false && probingData.state === 'n/d') {
    message = "monitoringSystemSuspendedMessage"
  } else if (probingData.state === 'offline' && !probingData.eserviceActive) {
    // ...
  }

  return (
    <Alert
      sx={{
        width: '100%',
      }}
      severity="warning"
    >
      {t(message)}
    </Alert>
  )
}


```